nsxt/resource_nsxt_manager_cluster.go

```diff
@@ -0,0 +1,459 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"strings"
+	"time"
+
+	"golang.org/x/exp/slices"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt-mp/nsx"
+	nsxModel "github.com/vmware/vsphere-automation-sdk-go/services/nsxt-mp/nsx/model"
+)
+
+var MaxGetClusterConfigRetry = 20
```

Comment:
I suggest we use parameters configured on provider, like `commonProviderConfig.MaxRetries`

Reply:
Got it, in that case we need to first convert the client from interface type to nsxtclients, then user clients.CommonConfig.MaxRetries